src/live.ipxe

```diff
@@ -9,6 +9,10 @@ item --gap Live Boot Distributions
 item live-debian ${space} Debian Live
 item live-fedora ${space} Fedora Live
 item live-ubuntu ${space} Ubuntu Live
+item live-kali ${space} Kali Live
```

Comment:
These items have been in alphabetical order should be like: 
live-debian
live-elementary
live-fedora
live-kali
live-q4os
live-tails
live-ubuntu

Also Please drop Tails, as is it should only be in development releases until we can figure out how to get networking working.

Reply:
There. I have sorted them by alphabetical order, and dropped Tails.